network-mux/network-mux.cabal

```diff
@@ -33,6 +33,8 @@ library
                        ieee754,
                        network,
                        process         >=1.6 && <1.7,
+                       statistics-linreg,
+                       vector,
```

Comment:
Yeah this is fairly heavy. We should indeed look at whether we can specialise.

Reply:
Is it ok to leave this until a later version (i.e before 1.2) - we need to decide the best regression model first.